scripts/fuzz.sh

```diff
@@ -103,18 +103,8 @@ main()
   # This also drops _leading_ '--'.
   shift $((OPTIND-1))
 
-  # Move this to a new fuzz.conf EXTRA_CONF_FILE if it grows bigger
-  local fuzz_configs=(
-    -DCONFIG_ZEPHYR_POSIX=y
-    -DCONFIG_ASSERT=y
-    -DCONFIG_EXCEPTION_DEBUG=y
-    -DCONFIG_ARCH_POSIX_TRAP_ON_FATAL=y
-    -DCONFIG_SYS_HEAP_BIG_ONLY=y
-    -DCONFIG_ZEPHYR_NATIVE_DRIVERS=y
-    -DCONFIG_ARCH_POSIX_LIBFUZZER=y
-    -DCONFIG_ZEPHYR_POSIX_FUZZ_TICKS=100
-    -DCONFIG_ASAN=y
-  )
+  # https://docs.zephyrproject.org/latest/build/kconfig/setting.html#initial-conf
+  local conf_files_list='prj.conf;boards/native_sim_libfuzzer.conf'
```

Comment:
can we make it so we can append to this list. I.e. the ASAN config would be split so we can use the base config with other sanitzer configs.

Reply:
Yes that's the idea. In some future PR(s) anyone can just go and (for instance) do:

```shell
if $something; then
   conf_files_list+=';debug.conf'
```